be/src/main/java/codesquad/issuetracker/dto/issue/IssueDto.java

```diff
@@ -33,12 +30,14 @@ public class IssueDto {
 
     private IssueDto() {}
 
-    public static IssueDto of(Issue issue, Milestone milestone, List<Member> assignees, List<Label> labels) {
+    public static IssueDto of(Issue issue) {
         Member writer = issue.getWriter();
-        return new IssueDto(issue.getId(), issue.getStatus(), issue.getSubject(), issue.getDescription(),
+        return new IssueDto(
+            issue.getId(), issue.getStatus(), issue.getSubject(), issue.getDescription(),
             writer.getIdentity(), writer.getProfileUrl(), issue.getCreatedDateTime(),
-            MilestoneMapper.convertToDto(milestone),
-            MemberMapper.convertToListDto(assignees),
-            LabelMapper.convertToListDto(labels));
+            MilestoneDto.convertToDto(issue.getMilestone()),
```

Comment:
팩토리 메소드 호출 과정에서 null 체크를 해줘도 좋을 것 같습니다.

```suggestion
            Milestone milesonte = issue.getMilestone();
            // ...
            
            milesonte != null ?  MilestoneDto.from(issue.getMilestone()) : null,
```